jobs/scripts/docs-styleguide.sh

```diff
@@ -11,5 +11,5 @@ sed -i 's/^five./#five./' sources.cfg
 sed -i 's/^z3c./#z3c./' sources.cfg
 sed -i 's/^Products.CMF/#Products.CMF/' sources.cfg
 sed -i 's/^Products.MailHost/#Products.MailHost/' sources.cfg
-pip install -r requirements.txt
-buildout -c docs.cfg install test
+pip install -Ur requirements.txt
+buildout buildout:git-clone-depth=1 install test
```

Comment:
This script is for running  the documentation styleguide, so we do need that ``docs.cfg`` that we create earlier, that or inline all the settings there in this buildout command, but without the ``auto-checkout = *`` it will be useless for checking the documentation.